src/newfight.cpp

```diff
@@ -305,6 +307,11 @@ struct melee_combat_data {
       if (cyber->cyberarms) {
         power += 2;
       }
+      
+      // Add +2 power to unarmed/melee, per MitS p147. -Vile
+      if (AFF_FLAGGED(ch, AFF_SPELL_FLAME_AURA)) {
```

Comment:
Since we're adding support for mob flame aura, let's change this to:
`if (AFF_FLAGGED(ch, AFF_SPELL_FLAME_AURA) || MOB_FLAGGED(ch, MOB_FLAMEAURA))`

Reply:
Done